<html>
<head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="description" content="" />
    <meta name="keywords" content="" />
    <!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
    <script src="/js/jquery.min.js"></script>
    <script src="/js/jquery.dropotron.min.js"></script>
    <script src="/js/skel.min.js"></script>
    <script src="/js/skel-layers.min.js"></script>
    <script src="/js/init.js"></script>
    <script src="/js/lightbox.js"></script>
    <link rel="stylesheet" href="/css/pygment.css"/>
    <noscript>
        <link rel="stylesheet" href="/css/skel.css" />
        <link rel="stylesheet" href="/css/style.css" />
        <link rel="stylesheet" href="/css/style-noscript.css" />
        <link href="/css/lightbox.css" rel="stylesheet"/>
    </noscript>
    <script src="//cdnjs.cloudflare.com/ajax/libs/modernizr/2.6.2/modernizr.min.js"></script>
    <!-- <link href="//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet"> -->
    <link rel="stylesheet" href="//cdn.materialdesignicons.com/1.3.41/css/materialdesignicons.min.css">
    <link  href="http://fonts.googleapis.com/css?family=Anonymous+Pro:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css" >
    <!--[if lte IE 8]><link rel="stylesheet" href="/css/ie/v8.css" /><![endif]-->
    <!--[if lte IE 9]><link rel="stylesheet" href="/css/ie/v9.css" /><![endif]-->
    <style>
    body, #skel-layers-wrapper {
    background: #CFF4EF;}
    </style>

    <!-- Favicon for all devices and sizes -->
    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="/apple-touch-icon-57x57.png" />
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="/apple-touch-icon-114x114.png" />
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="/apple-touch-icon-72x72.png" />
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144x144.png" />
    <link rel="apple-touch-icon-precomposed" sizes="60x60" href="/apple-touch-icon-60x60.png" />
    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="/apple-touch-icon-120x120.png" />
    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="/apple-touch-icon-76x76.png" />
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="/apple-touch-icon-152x152.png" />
    <link rel="icon" type="image/png" href="/favicon-196x196.png" sizes="196x196" />
    <link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96" />
    <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32" />
    <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16" />
    <link rel="icon" type="image/png" href="/favicon-128.png" sizes="128x128" />
    <meta name="application-name" content="&nbsp;"/>
    <meta name="msapplication-TileColor" content="#FFFFFF" />
    <meta name="msapplication-TileImage" content="/mstile-144x144.png" />
    <meta name="msapplication-square70x70logo" content="/mstile-70x70.png" />
    <meta name="msapplication-square150x150logo" content="/mstile-150x150.png" />
    <meta name="msapplication-wide310x150logo" content="/mstile-310x150.png" />
    <meta name="msapplication-square310x310logo" content="/mstile-310x310.png" />


    <title>Research | Yeo Lab </title>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width">
</head>

<body class=" loading">



    <!-- Header -->
    <header id="header" >
        <h1 class="logo">
            <a href="..">Yeo Lab</a>
        </h1>
        <nav id="nav">
            <ul>
                <!-- <li class="current"><a href="index.html">Welcome</a></li> -->
                    <li><a href="/research/">Research</a></li>
                    <li><a href="/papers/">Papers</a></li>
                    <li><a href="/funding/">Funding</a></li>
                    <li><a href="/software/">Software</a></li>
                    <li><a href="/links/">Links</a></li>
                <li><a href="/people" class="button special">People</a></li>
            </ul>
        </nav>
    </header>

<style>
.image:before{
  background-image: None;
}
.mdi {
  font-size: 9em;
  margin-top: -150px;
}

</style>


<!-- Main -->
<article id="main">

  <header class="special container">
    <span class="mdi mdi-magnify"></span>
    <h2>Research</h2>
    <!-- add page sub title here -->
    <!-- <p>Where that in the center faces the nameless horrors alone.</p> -->
  </header>

  <!-- One -->
  <section class="wrapper style4 container">

    <!-- Content -->
      <div class="content">
        <section>
          <!-- <a href="#" class="image feature"><img src="images/pic04.jpg" alt="" /></a> -->
          <p><style>
.image:before{
  background-image: None;
  height: 100%;
}
em{
  font-style: italic;
}
</style>

<div class="row">
  <div class="6u">

<section>
  <a href="#stem-cells" class="image feature"><img src="/images/research/IMP1_OCT4 IF_2.svg" alt="" /></a>
  <header>
    <h3>Homeostasis in pluripotent stem&nbsp;cells</h3>
  </header>
  <p>We study how <span class="caps">RNA</span> binding proteins and <span class="caps">RNA</span> modifications affect cellular homeostasis in human pluripotent stem&nbsp;cells.</p>
</section>

  </div>

<div class="6u">

<section>
  <a href="#neural-rbps" class="image feature"><img src="/images/research/neuron.svg" alt="" /></a>
  <header>
    <h3>Neural <span class="caps">RNA</span> binding&nbsp;proteins</h3>
  </header>
  <p>We are interested in how defects in <span class="caps">RNA</span> binding proteins cause neurological disease, such as <span class="caps">ALS</span>.</p>
</section>

<p></div>
</div>
<div class="row">
  <div class="6u"></p>
<section>
  <a href="#rna-processing" class="image feature"><img src="/images/research/mRNP.svg" alt="Decoding the relationships between RNAs, their binding partners, and the effect on the RNA life cycle"/></a>
  <header>
    <h3><span class="caps">RNA</span>&nbsp;Processing</h3>
  </header>
  <p>We study post-transcriptional processing of RNAs by multiple&nbsp;mechanisms.</p>
</section>

<p></div>
  <div class="6u"></p>
<section>
  <a href="#single-cell" class="image feature"><img src="/images/research/immunology_singlecell.svg" alt="" /></a>
  <header>
    <h3>Single-cell&nbsp;analyses</h3>
  </header>
  <p>We develop new computational approaches to decipher biological meaning from thousands of single-cell <span class="caps">RNA</span>-seq/proteomics&nbsp;data.</p>
</section>

<p></div>
</div></p>
<p>A major focus of our lab is to understand how gene expression is controlled at the <span class="caps">RNA</span> level to maintain proper functioning of cells during development and aging.  Over the past decade, there has been a dramatic increase in the recognition that members of a broad class of proteins termed <span class="caps">RNA</span> binding proteins (RBPs) are crucial for <strong>maintaining molecular and cellular homeostasis</strong>. RBPs regulate processes such as cell survival, pluripotency of embryonic stem cells, and neuronal function, as well as aid in the transition between cellular states in <strong>response to stimuli</strong>, such as during neural specification of stem cells, cellular stresses, or viral&nbsp;infections.</p>
<p>RBPs perform these tasks by influencing the fate of hundreds to thousands of target RNAs via interactions with specific sequences or <span class="caps">RNA</span> secondary structural features within the transcribed <span class="caps">RNA</span> molecule (Figure 1A) to regulate <strong>post-transcriptional processing of <span class="caps">RNA</span></strong> such as alternative splicing, polyadenylation, stability and turnover, localization, or translation control. Consequently, mutations or defects in the expression and/or localization of these RBPs (Figure 1B) cause a diversity of diseases such as neurodegeneration, cancer, and obesity. With the increasing number of proteins that are now classified as RBPs (more than a thousand) and the startling variation in diseases that they seem to cause, fundamental questions about how RBPs function, which <span class="caps">RNA</span> targets they interact with, and how they regulate gene expression remain unsolved.  In particular, addressing these in the context of development and genetic diseases is crucial to predicting consequences of abnormal behavior of these RBPs and identifying new therapeutics for human&nbsp;diseases.     </p>
<p>In our group, our primary aims have been to uncover regulatory principles of how RBPs influence <span class="caps">RNA</span> processing.&nbsp;Specifically:</p>
<ol>
<li>How do RBPs maintain cellular homeostasis in different cell-types and in&nbsp;development?</li>
<li>How do RBPs change in expression and respond to environmental stimuli (such as cellular stress and virus&nbsp;infection)?   </li>
<li>How do RBPs give rise to disease-relevant phenotypes when&nbsp;altered?</li>
</ol>
<p>In doing so, our secondary aims&nbsp;are:</p>
<ol>
<li>Developing molecular and chemical methods to alter <span class="caps">RBP</span>-<span class="caps">RNA</span> interactions to modulate gene&nbsp;expression.</li>
<li>Engineering novel protein-complexes that interact with <span class="caps">RNA</span> at specific sites to track and modulate <span class="caps">RNA</span>&nbsp;expression.</li>
<li>Developing computational and experimental workflows for understanding <span class="caps">RNA</span> processing at the single-cell&nbsp;level.</li>
</ol>
<p><strong>Our work strikes an important balance between hypothesis-driven research and exploration</strong>. In my lab, we apply a wide variety of approaches that include molecular biology, biochemistry, imaging, high-throughput sequencing, and large-scale genetic, genomics and chemical screens.  One of our key strengths is in our ability to analyze diverse and large datasets generated from high-throughput genomics/genetics approaches in a rigorous and reproducible manner. From our explorations, we have identified many novel <span class="caps">RNA</span> targets of <span class="caps">RNA</span> binding proteins and new mechanisms of regulation. This balance is a key point in our lab as it encourages trainees to keep an open scientific mind while still testing specific&nbsp;hypotheses.</p>
<h2>Fundamental insights from global&nbsp;perspectives.</h2>
<h3>Homeostasis in pluripotent stem cells <a name="stem-cells"></a></h3>
<p>Post-transcriptional regulation of gene expression had been proposed to be important for the survival and pluripotency of pluripotent stem cells (PSCs).  PSCs are an infinite resource for regenerative medicine and an exciting and accessible system for modeling human development and diseases in a dish.  My research in this area focuses on hypotheses underlying the importance of candidate RBPs with two characteristics, namely RBPs that are highly expressed in PSCs compared to other cell-types, and whose expression levels decrease when PSCs are differentiated towards mature&nbsp;lineages.</p>
<p><em><span class="caps">RBFOX2</span></em>. Our expectation was that alternative splicing changes in many genes would accompany the differentiation of PSCs to mature lineages.  However it was not known which are these genes and which <span class="caps">RBP</span> (splicing factor) would be responsible for alternative splicing changes.  To test the hypothesis that one or more RBPs are responsible for modulating these changes, we had performed an <em>unbiased identification of alternative splicing (<span class="caps">AS</span>) events during neural differentiation of human embryonic stem cells (hESCs) using splicing-sensitive microarrays</em> (Yeo et al.).  We found that the motif (U)<span class="caps">GCAUG</span> was enriched within introns proximal to alternative exons in hESCs.  Thus, we pursued the Rbfox family of RBPs that was known to bind to (U)<span class="caps">GCAUG</span> to regulate alternative splicing. Unsatisfied with selecting single genes that frequently lead to conclusions that do not generalize to the majority of <span class="caps">RNA</span> targets, we isolated direct <span class="caps">RNA</span> binding sites of <span class="caps">RBFOX2</span> across the transcriptome using my modification of the cross-linking and immunoprecipitation (<span class="caps">CLIP</span>) technology.  As one of the first lab to adapt the technology for high-throughput sequencing using the Illumina <span class="caps">HTS</span> platform (Yeo et al., 2009), we developed computational approaches to analyze and integrate global binding data with functional information such as splicing changes measured by <span class="caps">RNA</span>-seq or arrays upon depletion of RBPs using RNAi or antisense oligonucleotide technology. We published the <em>first genome-wide <span class="caps">RNA</span> binding mapping study in PSCs</em> that verified that Rbfox2 indeed binds in vivo to these predicted binding sites, and was one of the first using <span class="caps">CLIP</span>-seq to show that <em>RBPs had position-dependent effects on alternative splicing</em> (Yeo et al.). In our explorations, we also showed that Rbfox2 was a “master splicing regulator” as it controlled other splicing factors and was <strong>necessary for <span class="caps">PSC</span> survival but not other cell-types</strong>, illustrating a <strong>distinct cellular function for an <span class="caps">RBP</span> in PSCs</strong>. In ongoing research we are validating other modes of <span class="caps">RBFOX2</span> regulation of gene expression (see <span class="caps">II</span>) and studying <span class="caps">RBFOX</span> proteins in the brain (see&nbsp;below).</p>
<p><em><span class="caps">LIN28</span>-let7-<span class="caps">IMP</span> network</em>. The Lin28, Imp1, Imp3, Esrp1 and Esrp2 RBPs are highly expressed in PSCs.  Importantly, Lin28 is a reprogramming factor, raising my hypothesis that its mRNA targets will reveal networks that are key for pluripotency. We published the <em>first nucleotide-level <span class="caps">RNA</span> binding map of the pluripotency-associated <span class="caps">RBP</span> Lin28 and showed that <span class="caps">LIN28</span> binds within <span class="caps">GGAGA</span> motifs in loop structures in mRNAs, reminiscent but distinct from its interaction with let-7 microRNA precursors</em> (Wilbert et al.).  In ongoing research, we are investigating the Imp1, Imp2 and Imp3 proteins, which are part of the regulatory circuit consisting of Lin28 and let-7 microRNAs. Using genome-wide binding assays, we found that <span class="caps">IMP1</span>, <span class="caps">IMP2</span> and <span class="caps">LIN28</span> bind to largely overlapping target genes. Importantly, when <span class="caps">IMP1</span> levels are reduced, <span class="caps">IMP1</span> targets such as cellular adhesion genes decrease, causing hESCs to detach and undergo apoptosis.  Thus we have unveiled a novel function for these proteins in PSCs (Conway<em>, Van Nostrand</em> et al, in&nbsp;preparation).  </p>
<h3>Normal and mutant functions of RBPs in the central nervous system <a name="neural-rbps"></a></h3>
<p>Defects in <span class="caps">RNA</span> regulation in the nervous system lead to neurodevelopmental dysfunction (e.g. autism) and neurodegenerative diseases (e.g. <span class="caps">ALS</span>). The mechanisms by which RBPs cause neurological disease are diverse, and in several cases the molecular details continue to remain&nbsp;elusive.</p>
<p><em><span class="caps">RBFOX1</span>, <span class="caps">RBFOX2</span>, <span class="caps">RBFOX3</span>.</em> Rbfox proteins interact with proteins mutated in spinal cerebellar ataxia and individuals with mutations mapping to the Rbfox1 gene locus have mental retardation, epilepsy and autism. Animal models with knockout or knockdown of Rbfox protein expression show extensive defects in neuronal physiology. <strong>To test the hypothesis that alternative splicing targets of Rbfox proteins would reveal pathways important in autism</strong>, we mapped the Rbfox 1 and 2 transcriptome-wide binding sites in mouse brains. We were excited to find that loci associated with autism spectrum disorders such as the Neurexins, Neuroligins and Shank genes contained both Rbfox sites nearby alternatively spliced exons but interestingly, also far away from any exons (LovcWe et al.). With further exploration, we found that similarly to exon-proximal sites, distal sites bound by Rbfox1 and Rbfox2 contain evolutionarily conserved <span class="caps">GCAUG</span> sequences and are associated with splicing regulation upon modulation of Rbfox abundance. We showed that a conserved long-range <span class="caps">RNA</span>-<span class="caps">RNA</span> base-pairing interaction (an “<span class="caps">RNA</span> bridge”) facilitates Rbfox-mediated exon inclusion in the <span class="caps">ENAH</span> (enabled homology) gene, demonstrating, <em>a fundamental <span class="caps">RNA</span>-mediated mechanism for <span class="caps">AS</span> control by distally bound RBPs</em> <a href="/papers/2013/nsmb2699.pdf">(Lovci et al.)</a>.</p>
<p><em><span class="caps">ALS</span>-associated RBPs</em>. A paradigm shift in the pathogenesis of Amyotrophic Lateral Sclerosis (<span class="caps">ALS</span>) occurred when mutations in RBPs, in particular Tar <span class="caps">DNA</span>-binding protein 43 (or <span class="caps">TDP</span>-43) and (fused in sarcoma) <span class="caps">FUS</span>/<span class="caps">TLS</span>, were found in patients with <span class="caps">ALS</span>. To understand the normal functions of <span class="caps">TDP</span>-43 and <span class="caps">FUS</span>/<span class="caps">TLS</span> in the central nervous system, in collaboration with Don Cleveland’s laboratory at <span class="caps">UCSD</span>, we generated <em>genome-wide <span class="caps">RNA</span> binding sites of <span class="caps">TDP</span>-43 in vivo</em> (Polymenidou et al., 2011). A year later, also in collaboration with Don’s lab, we published <em>genome-wide binding sites for <span class="caps">FUS</span>/<span class="caps">TLS</span></em> (Lagier-Tourenne et al.).  Our results supported a common loss-of-function (gene expression and alternative splicing) pathway in the pathogenesis of <span class="caps">TDP</span>-43 and <span class="caps">FUS</span>/<span class="caps">TLS</span> misregulation.  In ongoing research in my lab, we are evaluating the role of <span class="caps">TAF15</span> and hnRNP protein A2/B1, reported to cause multisystem proteinopathy and <span class="caps">ALS</span> (Kim et&nbsp;al.).</p>
<h2>New mechanistic views of <span class="caps">RNA</span> processing <a name="rna-processing"></a></h2>
<p><em>Alternative splicing regulation by multiple RBPs</em>. In order for cells to maintain homeostasis, it is thought that multiple RBPs coordinately control <span class="caps">RNA</span> processing of individual transcripts. The specific hypothesis we sought to test was that multiple RBPs would coordinately control alternative splicing events in human cells. As a pilot experiment, we were interested in how members of the most abundantly expressed RBPs named heterogeneous ribonucleoparticle (hnRNP) proteins coordinately affect alternative splicing in human cells. We published the <em>first integrated genome-wide analyses of alternative splicing regulated by multiple RBPs and their binding sites in human cells</em> (Huelga et al.). Surprisingly, we showed that the hnRNPs had a high degree of cross- and auto-regulation, and basically attempted to compensate for each other’s&nbsp;absence.</p>
<p>To understand coordinated control it important to systematically evaluate what is the composition of <span class="caps">RBP</span>-<span class="caps">RNA</span> complexes? RBPs interact with shared <span class="caps">RNA</span> substrates to form <span class="caps">RNP</span>-complexes. Knowing the protein interactors of each <span class="caps">RBP</span> will enable us to attribute novel functions to known RBPs. In ongoing research, we performed <strong>unbiased quantitative proteomics</strong> to comprehensively identify proteins that interact with 10 hnRNPs, <span class="caps">RBFOX2</span> and <span class="caps">UPF1</span> proteins. We identified hundreds to thousands of interactors for each <span class="caps">RBP</span>, the majority of which <span class="caps">RNA</span>-dependent interactors. This protein-protein interaction data is a rich resource for identifying previously unknown linkages to cellular machineries, signaling pathways or subcellular&nbsp;structures.</p>
<p>And as part of the <span class="caps">NIH</span>-funded <span class="caps">ENCODE</span> efforts, my lab screened more than 700 commercially available antibodies against &gt;500 RBPs for their ability to selectively immunoprecipitate the target protein.  We found antibodies that are effective against &gt;250 RBPs where all this information is publicly available to the community. With these antibodies, my lab is in the process of generating a large <span class="caps">RBP</span>-<span class="caps">RNA</span> interactome dataset in the world to-date, consisting of 250 RBPs in two human cell-lines.  Our improvements in the cross-linking and immunoprecipitation (<span class="caps">CLIP</span>) protocol have enabled us to operate at this scale in a highly reproducible fashion, setting the standard for such analyses in the&nbsp;field.  </p>
<p><em>Adenosine-to-inosine <span class="caps">RNA</span> editing</em> is mediated by RBPs named adenosine deaminase acting on <span class="caps">RNA</span> (ADARs). With the advent of high-throughput sequencing, <span class="caps">RNA</span>-seq datasets can be mined for evidence of <span class="caps">RNA</span> editing in tissues and cell-lines.  However, it is often difficult to distinguish A-to-I <span class="caps">RNA</span> edit sites in the transcriptome with sequencing errors or nucleotide variation in the genomes. We started a project to build a computational approach that takes as input <span class="caps">RNA</span>-seq data and predicts A-to-I sites.  At that point, several high-profile papers were published in genome-wide identification of A-to-I sites, rapidly followed by criticisms in the community about the treatment of these data bordering on being incorrect.  Taking a step back, we realized that many of these studies did not have the proper genetic control, which is to knockout the <span class="caps">ADAR</span> proteins thus any predicted A-to-I site in the <span class="caps">ADAR</span> knockout would be considered a false positive by the algorithm.  In a close collaboration with Prof. Hundley (Indiana University), we found that the non-catalytic <span class="caps">ADR</span>-2 in worms could still affect <span class="caps">RNA</span> editing in vivo by affecting the same substrates as <span class="caps">ADR</span>-1 (Washburn et al Cell&nbsp;Reports).</p>
<p><em><span class="caps">RNA</span> quality control</em> mechanisms are important for the removal of faulty RNAs.  In order to discriminate normal RNAs from ones destined for degradation, Suzanne Lee that is co-mentored with Prof. Lykke-Andersen (<span class="caps">UCSD</span>) led a study that demonstrated that the ATPase cycle of the <span class="caps">SF1</span> helicase Upf1 is necessary for mRNA target discrimination. Gabriel Pratt and Fernando Martinez also contributed significantly to the study. Pratt performed all the bioinformatics analyzing the large <span class="caps">CLIP</span>-seq, <span class="caps">RIP</span>-seq and <span class="caps">RNA</span>-seq datasets generated by Fernando and Suzanne (Lee et al Molecular&nbsp;Cell).</p>
<p><em>Repetitive elements in the genome</em> can interact with RBPs. In collaboration with Tim Behrens at Genentech, we found that the Ro60 <span class="caps">RBP</span> binds an <span class="caps">RNA</span> motif derived from endogenous Alu retroelements. Alu transcripts were found to be induced by type I interferon and stimulated pro-inflammatory cytokine secretion by human peripheral blood cells.  Importantly Ro60 deletion resulted in enhanced expression of Alu RNAs and interferon-regulated genes.  Tiffany Hung, a fellow at Genentech who led the study went on to show that anti-Ro60 positive systemic lupus erythematosus immune complexes contained Alu RNAs and Alu transcripts were enriched in <span class="caps">SLE</span> whole blood samples compared to control.  This exciting finding was recently published in the journal Science in&nbsp;2015.</p>
<h2>Single cell views of <span class="caps">RNA</span> processing <a name="single-cell"></a></h2>
<p>Until recently, the overwhelming majority of biological conclusions about mechanisms of <span class="caps">RNA</span> processing and transcriptomic diversity are derived from bulk populations of cells or tissues.  However, the multitude of cell-types in heterogeneous mixtures in vivo and even in vitro during differentiation of PSCs, coupled with stochastic control of gene expression have been largely ignored. The transcriptomic composition of individual cells is lost in conventional sequencing projects, which typically analyze <span class="caps">RNA</span> extracted from pooled populations of cells and variations that specify individual cell-types are largely hidden in the bulk signal. Advances in the techniques for the isolation of single cells coupled with next-generation sequencing (<span class="caps">NGS</span>) devices have revealed new insights in specific “bimodal” gene expression patterns within a population of cells (Shalek et al., 2013), allele specific expression (Deng et al., 2014) and transcriptional programs in single cells from human and mouse embryos (Xue et al., 2013; Yan et al.,&nbsp;2013).</p>
<p>In ongoing research, my lab has developed novel experimental and computational analytical tools to isolate and analyze hundreds to thousands of individual cells. As part of our development and validation process for our tools, together with our collaborator John Chang (<span class="caps">UCSD</span>), we have analyzed ~5,000 single cells, each analyzed using qRT-<span class="caps">PCR</span> of ~90 genes to develop a fate map in cells, published recently in Nature Immunology (Arsenio et al.). My graduate student Boyko Kakaradov and Janilyn Arsenio, a talented postdoc in Chang’s lab co-led the&nbsp;work.</p></p>
        </section>
      </div>

  </section>

  <!-- Two -->
  <!--     <section class="wrapper style1 container special">
        <div class="row">
        </div>
    </section> -->

  </article>

<!-- Footer -->

<style>
#footer .mdi{
  font-size: 4em;
}
</style>

<footer id="footer">

    <ul class="icons">
        <li>
          <a href="http://github.com/yeolab" class="mdi mdi-github-circle"><span class="label"></span></a>
        </li>
        <li>
          <a href="http://twitter.com/yeo_lab" class="mdi mdi-twitter-circle"><span class="label"></span></a>
        </li>
    </ul>

    <span class="copyright">&copy; Yeo Lab. All rights reserved. Design: <a href="http://html5up.net">HTML5 UP</a>.</span>

</footer>
</body>
</html>